SLS - SCM GW Auth (#44967)
demisto/content: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake_description.md

1. In the War Room, run the command `!GetLicenseID` to get the `license ID`.
2. Go to __Settings__ > __ABOUT__ > __License__ to get the `Customer Name`.

> 3. Go to the [HUB](https://apps.paloaltonetworks.com/apps) and login using your **Palo Alto Networks** credentials.

Reviewer: ```suggestion
3. Go to the [HUB](https://apps.paloaltonetworks.com/apps) and log in using your Palo Alto Networks credentials.
```

4. Under the `Cortex™ XSOAR` app select the relevant instance. If you don't have an active `Cortex™ XSOAR` app check out the Hub [Docs site](https://docs.paloaltonetworks.com/hub/hub-getting-started) to learn about app activation.
5. Insert the `license ID` and the `Customer Name` in the required fields and complete the authentication process in order
to get the __Registration ID__, __Encryption Key__, and __Client Secret__.
6. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
7. Search for Palo Alto Networks Cortex v2.
8. Click __Add instance__ to create and configure a new integration instance.
    * __Name__: A textual name for the integration instance.

>     * __Registration ID__: From the authentication process

Reviewer: ```suggestion
    * __Registration ID__: From the authentication process.
```

       * The token retrieval URL is inferred based on the tenant's FedRAMP status unless explicitly specified in the __Registration ID__ parameter in the format `REGISTRATION_ID@URL`.

>     * __Encryption Key__: From the authentication process

Reviewer: ```suggestion
    * __Encryption Key__: From the authentication process.
```

>     * __Client Secret__: From the authentication process

Reviewer: ```suggestion
    * __Client Secret__: From the authentication process.
```

>     * __proxy__: Use system proxy settings

Reviewer: ```suggestion
    * __proxy__: Use system proxy settings.
```

>     * __insecure__: Trust any certificate (not secure)

Reviewer: ```suggestion
    * __insecure__: Trust any certificate (not secure).
```

>     * __Fetch incidents__: Whether to fetch incidents or not

Reviewer: ```suggestion
    * __Fetch incidents__: Whether to fetch incidents.
```

>     * __first_fetch_timestamp__: First fetch time (<number> <time unit>, e.g., 12 hours, 7 days, 3 months, 1 year)

Reviewer: ```suggestion
    * __first_fetch_timestamp__: First fetch time (<number> <time unit>, e.g., 12 hours, 7 days, 3 months, 1 year).
```

>     * __Severity of events to fetch (Firewall)__: Select from all,Critical,High,Medium,Low,Informational,Unused

Reviewer: ```suggestion
    * __Severity of events to fetch (Firewall)__: Select from all, Critical, High, Medium, Low, Informational, or Unused.
```

>     * __Subtype of events to fetch (Firewall)__: Select from all,attack,url,virus,spyware,vulnerability,file,scan,flood,packet,resource,data,url-content,wildfire,extpcap,wildfire-virus,http-hdr-insert,http-hdr,email-hdr,spyware-dns,spyware-wildfire-dns,spyware-wpc-dns,spyware-custom-dns,spyware-cloud-dns,spyware-raven,spyware-wildfire-raven,spyware-wpc-raven,wpc-virus,sctp

Reviewer: ```suggestion
    * __Subtype of events to fetch (Firewall)__: Select from all, attack, url, virus, spyware, vulnerability, file, scan, flood, packet, resource, data, url-content, wildfire, extpcap, wildfire-virus, http-hdr-insert, http-hdr, email-hdr, spyware-dns, spyware-wildfire-dns, spyware-wpc-dns, spyware-custom-dns, spyware-cloud-dns, spyware-raven, spyware-wildfire-raven, spyware-wpc-raven, or wpc-virus,sctp.
```

2. First 48 hours - once in 10 minutes.
3. After that every 60 minutes.

> If you wish to try authenticating again, run the 'cdl-reset-authentication-timeout' command and retry.

Reviewer: ```suggestion
To try authenticating again, run the 'cdl-reset-authentication-timeout' command and retry.
```
